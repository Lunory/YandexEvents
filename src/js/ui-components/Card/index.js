export { default as SmallCard } from './SmallCard'
export { default as MediumCard } from './MediumCard'
export { default as LargeCard } from './LargeCard'
