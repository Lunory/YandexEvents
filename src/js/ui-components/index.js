export { default as Avatar } from './Avatar'
export { default as Button } from './Button'
export { default as FloatingButton } from './FloatingButton'
export { default as Carousel } from './Carousel'
export { default as Image } from './Image'
export { default as Slider } from './Slider'
export { default as Container } from './Container'
export { default as Tabs } from './Tabs'
export { default as TopBar } from './TopBar'
export { default as Icon } from './Icon'
export { default as Card } from './Card'
export { default as SlideCard } from './SlideCard'
export { default as Spinner } from './Spinner'
export { default as MapCard } from './MapCard'
export { default as HolidayCard } from './HolidayCard'
export { default as BottomNavigation } from './BottomNavigation'
