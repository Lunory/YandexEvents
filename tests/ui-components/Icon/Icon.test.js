import React from 'react'
import renderer from 'react-test-renderer'

import { Icon } from 'ui-components'

describe('Icon', () => {
  it('Проверка снепшота компонента', () => {
    const component = renderer.create(
      <Icon type='star' />
    )

    expect(component.toJSON()).toMatchSnapshot()
  })
})
