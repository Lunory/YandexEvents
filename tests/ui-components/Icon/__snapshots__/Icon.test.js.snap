// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon Проверка снепшота компонента 1`] = `
<svg
  dangerouslySetInnerHTML={
    Object {
      "__html": "<g id=\\"Symbols\\" stroke=\\"none\\" stroke-width=\\"1\\" fill=\\"none\\" fill-rule=\\"evenodd\\" stroke-linecap=\\"round\\"><g id=\\"bottomnav_ya-events_now\\" transform=\\"translate(-167.000000, -8.000000)\\" stroke-width=\\"2\\" stroke=\\"#777777\\"><g id=\\"icon-all-events-disabled\\" transform=\\"translate(167.000000, 8.000000)\\"><polygon id=\\"Star-Copy\\" points=\\"12 18 6.12214748 21.0901699 7.24471742 14.545085 2.48943484 9.90983006 9.06107374 8.95491503 12 3 14.9389263 8.95491503 21.5105652 9.90983006 16.7552826 14.545085 17.8778525 21.0901699\\"></polygon></g></g></g>",
    }
  }
  fill="#000"
  height="24"
  viewBox="0 0 24 24"
  width="24"
/>
`;
